providers/implementations/serializers/libimplementations-lib-serializer_ecx.d.tmp

```diff
@@ -0,0 +1,25 @@
+providers/implementations/serializers/libimplementations-lib-serializer_ecx.o: \
```

Comment:
oops

Reply:
this is a good reason not to build in the source directory...  but also to fix .gitignore

Reply:
Fixed. Also fixed .gitignore in #11122